trace_replay/trace_replay.cc

```diff
@@ -208,8 +210,7 @@ Status Replayer::Replay() {
   WriteOptions woptions;
   ReadOptions roptions;
   Trace trace;
-  uint64_t ops = 0;
-  Iterator* single_iter = nullptr;
+  // uint64_t ops = 0;
```

Comment:
Why comment it out rather than remove it altogether?

Reply:
Good point. I have now removed it.